### ISSUE TITLE
Function return types are not sorted

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#44]
Input
```
export type Props = {
  callback2: () => "B" | "A",
  callback1(): "B" | "A",
}
```
Expected output
```
export type Props = {
  callback1(): "A" | "B",
  callback2: () => "A" | "B",
}
```
Actual output:
Same as input

**Comment by k2snowman69** [comment]
Looks like for Typescript this is TSMethodSignature and TSFunctionType which looks like you would push node.typeAnnotation
For Flow looks like `FunctionTypeAnnotation` and needing to push node.returnType.

**Comment by k2snowman69** [comment]
Fix and resolved in 2.1.6
